cdc/kv/client.go

```diff
@@ -294,7 +294,6 @@ type CDCKVClient interface {
 		ctx context.Context,
 		span regionspan.ComparableSpan,
 		ts uint64,
-		enableOldValue bool,
```

Comment:
why remove the enableOldValue flag in this PR?  Is there  an issue related to this item?

Reply:
The `enableOldValue ` param has been deprecated monthes ago, it's a cleanup for dead code.